Zend/zend_language_scanner.l

```diff
@@ -1496,9 +1496,8 @@ NEWLINE ("\r"|"\n"|"\r\n")
 
 <ST_IN_SCRIPTING>"("{TABS_AND_SPACES}"real"{TABS_AND_SPACES}")" {
 	if (PARSER_MODE()) {
-		zend_error(E_DEPRECATED, "The (real) cast is deprecated, use (float) instead");
+		zend_throw_exception(zend_ce_parse_error, "The (real) cast has been removed, use (float) instead", 0);
 	}
-	RETURN_TOKEN(T_DOUBLE_CAST);
```

Comment:
This line shouldn't be removed (this is the cause of the changes in the tokenizer tests).